loadable_apps/loadable_sample/Kconfig

```diff
@@ -3,17 +3,23 @@
 # see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
 #
 
-menuconfig EXAMPLES_ELF
-	bool "ELF Loader Example"
+config EXAMPLES_LOADABLE
+	bool "Enable Loadable Test Scenarios"
 	default n
-	select LIBC_EXECFUNCS
-	depends on ELF
+	depends on APP_BINARY_SEPARATION
 	---help---
-		Enable the ELF Loader Example
+		Enable Loadable Examples
 
-if EXAMPLES_ELF
+if EXAMPLES_LOADABLE
+config EXAMPLES_MANUAL_TEST
```

Comment:
Let's rename it as `EXAMPLES_LOADABLE_MANUAL_TEST`.
But Is this config necessary? Usually all of others are manual tests and when we want to run repeately, we enable some other config.